schema/codelists/entityType.csv

```diff
@@ -1,6 +1,8 @@
 code,title,description,technical note
 registeredEntity,Registered Entity,"Any legal entity created through an act of official registration. In most cases, registered entities will have an official issued identifier.",
-legalEntity,Legal entity,"A body with distinct legal personality, such as a government department or international institution, but which is not otherwise uniquely identified in some official register.",
```

Comment:
@ScatteredInk - I removed, 'such as a government department' from the legalEntity definition. But maybe the addition of the bolded words below would add more distinction:

Legal entity,"A body with distinct legal personality, such as an international institution **or statutory corporation**, but which is not otherwise uniquely identified in some official register."

State body,A **core** administrative or legislative unit within a state's apparatus.,

I think that helps a bit? (There will always be edge cases that sit on the boundary between definitions. But we should aim to make that a narrow boundary!)

FWIW I'd say the Oil & Pipelines Agency is a legal entity.

What do you think?

Reply:
Yes, that sounds good to me.